cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -421,3 +424,91 @@ def test_service_infra_tenant_quota_with_default_entry_point(request, appliance,
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert provision_request.row.reason.text == "Quota Exceeded"
+
+
+@pytest.fixture
+def configure_mail(domain):
+    """This fixture copies email instance to custom domain"""
+    approver = fauxfactory.gen_email()
+    default_recipient = fauxfactory.gen_email()
+    from_user = fauxfactory.gen_email()
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="Configuration"
+    ).classes.instantiate(name="Email").instances.instantiate(name="Default").copy_to(domain.name)
+    instance = (
+        domain.namespaces.instantiate(name="Configuration").classes.instantiate(
+            name="Email")).instances.instantiate(name="Default")
+    with update(instance):
+        instance.fields = {
+            "approver": {
+                "value": approver
+            },
+            "default_recipient": {
+                "value": default_recipient
+            },
+            "from": {
+                "value": from_user
+            },
+        }
+    yield approver, default_recipient, from_user
+
+
+@pytest.mark.meta(automates=[1579031])
+@pytest.mark.tier(1)
+@pytest.mark.provider([RHEVMProvider], override=True, selector=ONE)
+@pytest.mark.parametrize(
+    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg'],
+    [
+        [('memory', '2'), {'hardware': {'memory': '4096'}}, ''],
+    ],
+    indirect=['set_roottenant_quota'],
+    ids=['max_memory']
+)
+def test_quota_exceed_mail_with_more_info_link(configure_mail, appliance, provider,
+                                               set_roottenant_quota, custom_prov_data, prov_data,
+                                               extra_msg, vm_name, template_name):
+    """
+    Bugzilla:
+        1579031
+        1759123
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/12h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.9
+        casecomponent: Infra
+        tags: quota
+        testSteps:
+            1. Copy instance ManageIQ/Configuration/Email/Default to custom domain
+            2. Enter values for fields: approver, default_recipient, from and signature
+            3. Provide valid mail address while provisioning Vm to exceed quota
+        expectedResults:
+            1. Custom domain available for email configuration
+            2. Updated fields with valid values
+            3. Quota exceed mail should be sent
+    """
+    approver, default_recipient, from_user = configure_mail
+    mail_to = fauxfactory.gen_email()
+    prov_data.update(custom_prov_data)
+    prov_data['catalog']['vm_name'] = vm_name
+    with LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[
```

Comment:
Hi @izapolsk 
- I had tried with/without fixture ```smtp_test``` and checked if I am getting any logs under log/emails folder. But I can see empty folders available. 
- Please let me know, if I am missing something in this test case.

Reply:
@ganeshhubale can you add this task as an improvement to the test? I agree with @izapolsk but would like to merge what you've got so far for automation coverage.

Reply:
@ganeshhubale, this fixture is used in a lot of tests. there are a lot of usage examples in our repo. if it doesn't work in your case, this means that you either use it wrongly or there is a bug in CFME and it doesn't send email.

please note: it will fail in PRT. there is such limitation.
if you need some help with it, please ping me in chat.